index.html

```diff
@@ -0,0 +1,1156 @@
+            </div>
+            <div class="obj_top_nav_black">
+              <a href="https://tq.co/">
+                Spaces
+              </a>
+            </div>
+            <div class="obj_top_nav_black">
+              <a href="https://thenextweb.com/terms-and-conditions/">
+                Terms & Conditions
+              </a>
+            </div>
+          </div>
+          <div class="right_items">
+            <div class="obj_top_nav_black">
+              <a href="https://thenextweb.com/about">
+                About
+              </a>
+            </div>
+            <div class="obj_top_nav_black">
+              <a href="https://thenextweb.com/advertise">
+                Advertise
+              </a>
+            </div>
+            <div class="obj_top_nav_black">
+              <a href="https://thenextweb.homerun.co/">
+                Jobs
+              </a>
+            </div>
+            <div class="obj_top_nav_black">
+              <a href="https://thenextweb.com/about/#contact">
+                Contact
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="search_nav">
+          <div class="left_items_search_nav">
+            <div class="obj_search_nav">
+              <a href="https://thenextweb.com/latest/">LATEST</a>
+            </div>
+            <div class="obj_search_nav">
+              <a href="https://thenextweb.com/hardfork/">HARD WORK</a>
+            </div>
+            <div class="obj_search_nav">
+              <a href="https://thenextweb.com/plugged">PLUGGED</a>
+            </div>
+            <div class="obj_search_nav">
+              <a href="https://thenextweb.com/adobe-fundamentals/">
+                FUNDAMENTALS
+              </a>
+            </div>
+            <div class="obj_search_nav">
+              <a href="https://thenextweb.com/readme/">README</a>
+            </div>
+            <div class="obj_search_nav">
+              <a href="https://thenextweb.com/growth-quarters">
+                GROWTH QUARTERS
+              </a>
+            </div>
+            <div class="obj_search_nav">
+              <a href="https://thenextweb.com/neural">NEURAL</a>
+            </div>
+          </div>
+          <div class="right_items_search_nav">
+            <a href="#" class="search_border">
+              <span class="search_nav_icon"></span>
+            </a>
+            <a href="#" class="hamburguer_border">
+              <span class="hamburger_menu">
+                <span class="line_hamburguer"></span>
+                <span class="line_hamburguer"></span>
+                <span class="line_hamburguer"></span>
+              </span>
+            </a>
+          </div>
+        </div>
+      </nav>
+      <div class="ad_nav">
+        <p>
+          <a href="https://thenextweb.com/conference">
+            Did you know we have a TECH FESTIVAL? Check it out -->
+          </a>
+        </p>
+      </div>
+    </header>
+
+    <main>
```

Comment:
Very good use of semantic tags in the documents! :clap:

Reply:
This is an almost perfect implementation and I am aware it is hard to find some bugs. But kindly find some bugs/errors (I left some hints below) and explain why you are requesting those changes with a polite-professional language 👌